src/game.html

```diff
@@ -8,20 +8,45 @@
 	</head>
 
 	<body>
-		<div>
-			<label for="name-label">Enter your name here: </label>
+		<div id="setup-screen">
+			<label for="name-label" data-translate="EnterName">Enter your name here: </label>
 			<input type="text" id="name" />
+
+			<div class="avatar-group">
+				<label for="avatar" data-translate="SelectAvatar">Select your avatar: </label>
+				<select id="avatar" name="avatar">
+					<option value="avatar1" data-translate="Av1">Avatar1</option>
+					<option value="avatar2" data-translate="Av2">Avatar2</option>
+					<option value="avatar3" data-translate="Av3">Avatar3</option>
+				</select>
+			</div>
+			<div><button type="button" id="start-button" onclick="start()" data-translate="Start">START</button></div>
 		</div>
-		<div class="avatar-group">
-			<label for="avatar">Select your avatar: </label>
-			<select id="avatar" name="avatar">
-				<option value="avatar1">Avatar1</option>
-				<option value="avatar2">Avatar2</option>
-				<option value="avatar3">Avatar3</option>
-			</select>
+		<div id="game-screen" style="display: none">
```

Comment:
Nu vrem sa folosim _style_ in-line. Pune o clasa mai bine care face display: none;